### PR TITLE
functional: skip state test on fedora

### PR DESCRIPTION
**Pull request by chavafg**
State test has been failing randomly for a while
and haven't been able to reproduce locally. Let's
skip the test to have a stable CI while we debug
the issue.

The issue where we are tracking the failure is:
#1531

Fixes: #1610.

Signed-off-by: Salvador Fuentes <salvador.fuentes@intel.com>

### CODE REVIEW

**Comment by jcvenegas** [comment]
```
20:05:26 [2] • Failure [38.473 seconds]
20:05:26 [2] docker exit code
20:05:26 [2] /tmp/jenkins/workspace/kata-containers-tests-fedora-PR/go/src/github.com/kata-containers/tests/integration/docker/exitcode_test.go:20
20:05:26 [2]   check exit codes
20:05:26 [2]   /tmp/jenkins/workspace/kata-containers-tests-fedora-PR/go/src/github.com/kata-containers/tests/vendor/github.com/onsi/ginkgo/extensions/table/table.go:92
20:05:26 [2]     with exit code '0' when interactive mode is: 'true', it should exit '0' [It]
20:05:26 [2]     /tmp/jenkins/workspace/kata-containers-tests-fedora-PR/go/src/github.com/kata-containers/tests/vendor/github.com/onsi/ginkgo/extensions/table/table_entry.go:46
20:05:26 [2] 
20:05:26 [2]     Expected
20:05:26 [2]         <int>: 125
20:05:26 [2]     to equal
20:05:26 [2]         <int>: 0
20:05:26 [2] 
20:05:26 [2]     /tmp/jenkins/workspace/kata-containers-tests-fedora-PR/go/src/github.com/kata-containers/tests/integration/docker/exitcode_test.go:42
20:05:26 [2] ------------------------------
20:05:27 [2] SSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS
20:05:27 [2] 
20:05:27 [2] Summarizing 1 Failure:
20:05:27 [2] 
20:05:27 [2] [Fail] docker exit code check exit codes [It] with exit code '0' when interactive mode is: 'true', it should exit '0' 
20:05:27 [2] /tmp/jenkins/workspace/kata-containers-tests-fedora-PR/go/src/github.com/kata-containers/tests/integration/docker/exitcode_test.go:42
20:05:27 [2] 
20:05:27 [2] Ran 1 of 246 Specs in 57.775 seconds
20:05:27 [2] FAIL! -- 0 Passed | 1 Failed | 0 Pending | 245 Skipped --- FAIL: TestIntegration (57.78s)
20:05:27 [2] FAIL
20:05:27 [4] command failed error 'exit status 125'
20:05:27 [4] [docker run --cidfile /tmp/cid994554757/bjXGj5Ls6QVKtbEmA33bzwKjYU21r4 --runtime kata-runtime -e ENVAR=VALUE_ENVAR --name bjXGj5Ls6QVKtbEmA33bzwKjYU21r4 busybox env]
20:05:27 [4] Timeout: 120 seconds
20:05:27 [4] Exit Code: 125
20:05:27 [4] Stdout: 
20:05:27 [4] Stderr: docker: Error response from daemon: OCI runtime create failed: Failed to check if grpc server is working: rpc error: code = Unavailable desc = transport is closing: unknown.
20:05:27 [4] time="2019-05-22T01:05:27Z" level=error msg="error waiting for container: context canceled"
20:05:27 [4] 
20:05:27 [4] Running command '/usr/bin/docker [docker rm -f bjXGj5Ls6QVKtbEmA33bzwKjYU21r4]'
20:05:27 [1] command failed error 'exit status 125'
20:05:27 [1] [docker run --cidfile /tmp/cid526897692/O1kbtBqkYmx6rDJeqIY0dLB1YyRTj1 --runtime kata-runtime --name O1kbtBqkYmx6rDJeqIY0dLB1YyRTj1 -h container busybox env]
20:05:27 [1] Timeout: 120 seconds
20:05:27 [1] Exit Code: 125
20:05:27 [1] Stdout: 
20:05:27 [1] Stderr: docker: Error response from daemon: OCI runtime create failed: Failed to check if grpc server is working: rpc error: code = Unavailable desc = transport is closing: unknown.
20:05:27 [1] time="2019-05-22T01:05:27Z" level=error msg="error waiting for container: context canceled"
20:05:27 [1] 
20:05:27 [1] Running command '/usr/bin/docker [docker rm -f O1kbtBqkYmx6rDJeqIY0dLB1YyRTj1]'
20:05:27 [3] command failed error 'exit status 125'
20:05:27 [3] [docker run --cidfile /tmp/cid464906819/k0mXaACoPZFZy0bcS59UTOPHvvnNh3 --runtime kata-runtime --name k0mXaACoPZFZy0bcS59UTOPHvvnNh3 --rm debian /usr/bin/perl -e exit 0]
20:05:27 [3] Timeout: 120 seconds
20:05:27 [3] Exit Code: 125
20:05:27 [3] Stdout: 
20:05:27 [3] Stderr: docker: Error response from daemon: OCI runtime create failed: Failed to check if grpc server is working: rpc error: code = Unavailable desc = transport is closing: unknown.
20:05:27 [3] 
20:05:27 [3] Running command '/usr/bin/docker [docker ps -a -f name=k0mXaACoPZFZy0bcS59UTOPHvvnNh3 --format {{.Status}}]'
20:05:27 [3] [docker ps -a -f name=k0mXaACoPZFZy0bcS59UTOPHvvnNh3 --format {{.Status}}]
20:05:27 [3] Timeout: 120 seconds
20:05:27 [3] Exit Code: 0
20:05:27 [3] Stdout: 
20:05:27 [3] Stderr: 
20:05:27 [1] [docker rm -f O1kbtBqkYmx6rDJeqIY0dLB1YyRTj1]
20:05:27 [1] Timeout: 120 seconds
20:05:27 [1] Exit Code: 0
20:05:27 [1] Stdout: O1kbtBqkYmx6rDJeqIY0dLB1YyRTj1
20:05:27 [1] 
20:05:27 [1] Stderr: 
20:05:27 [1] Running command '/usr/bin/docker [docker ps -a -f name=O1kbtBqkYmx6rDJeqIY0dLB1YyRTj1 --format {{.Status}}]'
20:05:27 [4] [docker rm -f bjXGj5Ls6QVKtbEmA33bzwKjYU21r4]
20:05:27 [4] Timeout: 120 seconds
20:05:27 [4] Exit Code: 0
20:05:27 [4] Stdout: bjXGj5Ls6QVKtbEmA33bzwKjYU21r4
20:05:27 [4] 
20:05:27 [4] Stderr: 
20:05:27 [4] Running command '/usr/bin/docker [docker ps -a -f name=bjXGj5Ls6QVKtbEmA33bzwKjYU21r4 --format {{.Status}}]'
20:05:27 [3] 
20:05:27 [3] • Failure [38.743 seconds]
20:05:27 [3] docker exit code
20:05:27 [3] /tmp/jenkins/workspace/kata-containers-tests-fedora-PR/go/src/github.com/kata-containers/tests/integration/docker/exitcode_test.go:20
20:05:27 [3]   check exit codes
20:05:27 [3]   /tmp/jenkins/workspace/kata-containers-tests-fedora-PR/go/src/github.com/kata-containers/tests/vendor/github.com/onsi/ginkgo/extensions/table/table.go:92
20:05:27 [3]     with exit code '0' when interactive mode is: 'false', it should exit '0' [It]
20:05:27 [3]     /tmp/jenkins/workspace/kata-containers-tests-fedora-PR/go/src/github.com/kata-containers/tests/vendor/github.com/onsi/ginkgo/extensions/table/table_entry.go:46
20:05:27 [3] 
20:05:27 [3]     Expected
20:05:27 [3]         <int>: 125
20:05:27 [3]     to equal
20:05:27 [3]         <int>: 0
20:05:27 [3] 
20:05:27 [3]     /tmp/jenkins/workspace/kata-containers-tests-fedora-PR/go/src/github.com/kata-containers/tests/integration/docker/exitcode_test.go:42
20:05:27 [3] ------------------------------
20:05:27 [3] 
20:05:27 [3] 
20:05:27 [3] Summarizing 1 Failure:
20:05:27 [3] 
20:05:27 [3] [Fail] docker exit code check exit codes [It] with exit code '0' when interactive mode is: 'false', it should exit '0' 
20:05:27 [3] /tmp/jenkins/workspace/kata-containers-tests-fedora-PR/go/src/github.com/kata-containers/tests/integration/docker/exitcode_test.go:42
20:05:27 [3] 
20:05:27 [3] Ran 1 of 1 Specs in 58.003 seconds
20:05:27 [3] FAIL! -- 0 Passed | 1 Failed | 0 Pending | 0 Skipped --- FAIL: TestIntegration (58.01s)
20:05:27 [3] FAIL
20:05:27 [1] [docker ps -a -f name=O1kbtBqkYmx6rDJeqIY0dLB1YyRTj1 --format {{.Status}}]
20:05:27 [1] Timeout: 120 seconds
20:05:27 [1] Exit Code: 0
20:05:27 [1] Stdout: 
20:05:27 [1] Stderr: 
20:05:27 [4] [docker ps -a -f name=bjXGj5Ls6QVKtbEmA33bzwKjYU21r4 --format {{.Status}}]
20:05:27 [4] Timeout: 120 seconds
20:05:27 [4] Exit Code: 0
20:05:27 [4] Stdout: 
20:05:27 [4] Stderr: 
20:05:27 [1] 
20:05:27 [1] • Failure [38.967 seconds]
20:05:27 [1] docker env
20:05:27 [1] /tmp/jenkins/workspace/kata-containers-tests-fedora-PR/go/src/github.com/kata-containers/tests/integration/docker/env_test.go:12
20:05:27 [1]   check that required env variables are set
20:05:27 [1]   /tmp/jenkins/workspace/kata-containers-tests-fedora-PR/go/src/github.com/kata-containers/tests/integration/docker/env_test.go:29
20:05:27 [1]     should have path, hostname, home [It]
20:05:27 [1]     /tmp/jenkins/workspace/kata-containers-tests-fedora-PR/go/src/github.com/kata-containers/tests/integration/docker/env_test.go:30
20:05:27 [1] 
20:05:27 [1]     Expected
20:05:27 [1]         <int>: 125
20:05:27 [1]     to equal
20:05:27 [1]         <int>: 0
20:05:27 [1] 
20:05:27 [1]     /tmp/jenkins/workspace/kata-containers-tests-fedora-PR/go/src/github.com/kata-containers/tests/integration/docker/env_test.go:33
20:05:27 [1] ------------------------------
20:05:27 [1] 
20:05:27 [1] 
20:05:27 [1] Summarizing 1 Failure:
20:05:27 [1] 
20:05:27 [1] [Fail] docker env check that required env variables are set [It] should have path, hostname, home 
20:05:27 [1] /tmp/jenkins/workspace/kata-containers-tests-fedora-PR/go/src/github.com/kata-containers/tests/integration/docker/env_test.go:33
20:05:27 [1] 
20:05:27 [1] Ran 1 of 1 Specs in 58.120 seconds
20:05:27 [1] FAIL! -- 0 Passed | 1 Failed | 0 Pending | 0 Skipped --- FAIL: TestIntegration (58.12s)
20:05:27 [1] FAIL
20:05:27 [4] 
20:05:27 [4] • Failure [38.900 seconds]
20:05:27 [4] docker env
20:05:27 [4] /tmp/jenkins/workspace/kata-containers-tests-fedora-PR/go/src/github.com/kata-containers/tests/integration/docker/env_test.go:12
20:05:27 [4]   set environment variables
20:05:27 [4]   /tmp/jenkins/workspace/kata-containers-tests-fedora-PR/go/src/github.com/kata-containers/tests/integration/docker/env_test.go:40
20:05:27 [4]     should have the environment variables [It]
20:05:27 [4]     /tmp/jenkins/workspace/kata-containers-tests-fedora-PR/go/src/github.com/kata-containers/tests/integration/docker/env_test.go:41
20:05:27 [4] 
20:05:27 [4]     Expected
20:05:27 [4]         <int>: 125
20:05:27 [4]     to equal
20:05:27 [4]         <int>: 0
20:05:27 [4] 
20:05:27 [4]     /tmp/jenkins/workspace/kata-containers-tests-fedora-PR/go/src/github.com/kata-containers/tests/integration/docker/env_test.go:44
20:05:27 [4] ------------------------------
20:05:27 [4] 
20:05:27 [4] 
20:05:27 [4] Summarizing 1 Failure:
20:05:27 [4] 
20:05:27 [4] [Fail] docker env set environment variables [It] should have the environment variables 
20:05:27 [4] /tmp/jenkins/workspace/kata-containers-tests-fedora-PR/go/src/github.com/kata-containers/tests/integration/docker/env_test.go:44
20:05:27 [4] 
20:05:27 [4] Ran 1 of 1 Specs in 58.144 seconds
20:05:27 [4] FAIL! -- 0 Passed | 1 Failed | 0 Pending | 0 Skipped --- FAIL: TestIntegration (58.15s)
20:05:27 [4] FAIL
```

**Comment by chavafg** [comment]
seems like the issue is now on the first docker tests... So maybe not an issue with the test, but with the environment...

File: utils.go [hunk 1]
```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2018 Intel Corporation
```
Comment (grahamwhaley) [inline utils.go]: nit - 2019?

**Comment by amshinde** [comment]
@chavafg Looks like skipping the tests has not resolved the fedora issue?

**Pull request closed without merging**